airflow/providers/snowflake/hooks/sql_api_generate_jwt.py

```diff
@@ -0,0 +1,174 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is not a hook.
it looks more like util and should be in util path

Reply:
Yes, that makes sense. I'll move it. thanks for pointing it out :)